app/views/groups/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<p id="notice"><%= notice %></p>
+
+<div class="container-fluid">
+  <div class="row offset-2">
+    <h1>Your Groups</h1>
+  </div>
+</div>    
+
+
+<div class="container-fluid">
+  <div class="row offset-2">
+  <%= link_to 'New Group', new_group_path %> 
+  </div>
+</div>
+<br>
+
+<div class="container-fluid">
+  <div class="row justify-content-center">
+    <div class="col-md-8">
+      <div class="card-columns">
+        <% @groups.each do |group| %>
+            <td><%= render "group_card", group: group %></td>
+        <% end %> 
```

Comment:
I don't know why, but collection does not seem to recognize each variable. If I use it, this error appears: 

"undefined local variable or method `group' for #<ActionView::Base:0x00000000010720>
Did you mean?  @groups"

Reply:
@agustinvd10 my mistake! Since the name of the partial doesn't match the model, you need to [specify the name of the local variable](https://guides.rubyonrails.org/layouts_and_rendering.html#local-variables)

Try
```erb
<td><%= render partial: "group_card", collection: @groups, as: :group %></td>
```